pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
 > More information: <https://www.gnu.org/software/coreutils/chmod>.
 
-- Give the [u]ser who owns a file the right to e[x]ecute it:
+- Add [r]ead/[w]rite/e[x]ecute permissions (`{{r|w|x}}` is referred later as `{{permissions}}`) for a file/directory owner:
 
-`chmod u+x {{file}}`
+`chmod +{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give the [u]ser rights to [r]ead and [w]rite to a file/directory:
+- Add specific permissions for an owner/users in [g]roup/[o]ther users/[a]ll at once (`{{u|g|o|a}}` is referred later as `{{users}}`):
 
-`chmod u+rw {{file_or_directory}}`
+`chmod {{u|g|o|a}}+{{permissions}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Remove e[x]ecutable rights from the [g]roup:
+- Add/remove/replace specific permissions:
 
-`chmod g-x {{file}}`
+`chmod {{users}}{{+|-|=}}{{permissions}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

Comment:
```suggestion
`chmod {{u|g|o|a}}{{+|-|=}}{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

Reply:
I've used `chmod {{u}}{{+|-|=}}{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}` instead.